ui/src/adapters/env.ts

```diff
@@ -0,0 +1,46 @@
+import { z } from "zod";
+
+import * as domain from "src/domain";
```

Comment:
We should avoid wild card imports and call either the interface here or the one in `domain` something else, or reassign it locally.

Reply:
> We should avoid wild card imports and call the different types of `Env` that are declared differently.

Why? The namespace is precisely there to provide the context.

> They're exported and should be addressable uniquely

They are addressable uniquely when properly namespaced.

> to avoid confusion and inconsistencies

Can you extend on what confusion and inconsistencies you see with the namespaces?

Reply:
A developer can write `import * as foo from "src/domain";`. It goes against the spirit of the linter enforcing named exports, since it leaves ambiguity.

Further, it makes it difficult to know which `Env` I am looking for and reliant on the IDE for hints. Lastly, we don't do this anywhere else, everything is a uniquely named export.

Reply:
> A developer can write `import * as foo from "src/domain";`.

I don't see any problem with this, it's up to the developer to decide the proper namespace in the context of the importing module, but ok will change it. It's not a big deal.

Reply:
Changed

Reply:
Hmm, let's change it to only import what we want. I think we should remain consistent and strict with naming.

Reply:
Can we also change it in `schemas.ts`? Just to remain consistent and also strict in naming, importing only what we use.

Reply:
Not in this PR. This is a massive change that would imply a lot of interface renames, plus I don't agree with the change and would like to discuss it first in the context of a dedicated issue. Does it sound fair?